spec/requests/api/v2/site_statuses_spec.rb

```diff
@@ -0,0 +1,122 @@
+require 'rails_helper'
+
+describe 'Site Helpers API V2' do
+  let(:user) { create(:user).tap { |u| organisation.users << u } }
+  let(:organisation) { site_status.course.provider.organisations.first }
+  let(:payload) { { email: user.email } }
+  let(:token) do
+    JWT.encode payload,
+                Settings.authentication.secret,
+                Settings.authentication.algorithm
+  end
+  let(:credentials) do
+    ActionController::HttpAuthentication::Token.encode_credentials(token)
+  end
+  let(:site_status) { create :site_status }
+  let(:params)      { {} }
+  let(:perform_request) do
+    patch(
+      api_v2_site_status_path(site_status),
+      headers: { 'HTTP_AUTHORIZATION' => credentials },
+      params: params
+    )
+  end
```

Comment:
I think this should be a `def`, we're not relying on the returned value from this `patch` are we?